packages/apollo/src/commands/service/check.ts

```diff
@@ -429,6 +422,19 @@ export default class ServiceCheck extends ProjectCommand {
                   this.error(
                     federatedServiceCompositionUnsuccessfulErrorMessage
                   );
+                } else {
+                  if (!checkSchemaResult) {
+                    throw new Error(
+                      "Violated invariant. Schema should have been validated against operations if" +
+                        "there were no composition errors"
+                    );
+                  }
+
+                  // this is used for the printing
+                  taskOutput.checkSchemaResult = checkSchemaResult;
```

Comment:
@zionts the only thing missing here was that the `checkSchemaResult` wasn't being attached to the task output, so the `formatHumanReadable` function below didn't have access to the data it expected. This should fix that :)